### DISCUSSION TITLE
2022/02/07/%E5%8F%AF%E5%8F%98%E5%8F%82%E6%95%B0/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#26] [General]
# 2022/02/07/%E5%8F%AF%E5%8F%98%E5%8F%82%E6%95%B0/

从 scanf() 和 printf() 讲起 在使用 scanf() 和 printf() 时，你会发现一个奇怪的现象，它的参数个数是不确定的，格式化字符串中有多少个%，后面就要跟多少个参数 1234printf(&quot;%d&quot;,a);printf(&quot;%d %d&quot;

https://nickxu.me/2022/02/07/%E5%8F%AF%E5%8F%98%E5%8F%82%E6%95%B0/

**Comment by illyber** [comment]
样例的那个标识符法，循环相加的时候，是不是多加了个tmp变量？
